lib/jsdom/living/nodes/HTMLFormElement-impl.js

```diff
@@ -84,6 +77,23 @@ class HTMLFormElementImpl extends HTMLElementImpl {
     notImplemented("HTMLFormElement.prototype.submit", this._ownerDocument._defaultView);
   }
 
+  requestSubmit(submitter = undefined) {
+    if (submitter !== undefined) {
+      if (!isSubmitButton(submitter)) {
+        throw new TypeError("The specified element is not a submit button");
+      }
+      if (submitter.form !== this) {
+        throw new DOMException("The specified element is not owned by this form element", "NotFoundError");
+      }
+    }
+
+    if (!fireAnEvent("submit", this, undefined, { bubbles: true, cancelable: true })) {
+      return;
+    }
+
+    notImplemented("HTMLFormElement.prototype.submit", this._ownerDocument._defaultView);
```

Comment:
Probably should be requestSubmit in the not-implemented message?

Reply:
Oops, fixed.